### ISSUE TITLE
Sourcemap support

### ISSUE THREAD

**Issue by benmccann** [xiphux/svimg#31]
It looks like the preprocess edits the `.svelte` files without updating the source map, which will break source maps

**Comment by xiphux** [comment]
Yes, I think sourcemap support was added sometime after this package was created.
